ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/boot.j2

```diff
@@ -0,0 +1,11 @@
+{# eos - System Boot Configuration #}
+{% if boot is arista.avd.defined %}
+!
+{%     if boot.secret.sha512_password is arista.avd.defined %}
+! boot secret sha512 {{ boot.secret.sha512_password }}
```

Comment:
`!` first here??

Reply:
Yes, that's what I saw on eos device when I ran ```show run```

Reply:
Correct, I tested it now. Got it what you were saying.